SCAutolib/models/card.py

```diff
@@ -161,10 +183,18 @@ def __exit__(self, exp_type, exp_value, exp_traceback):
 
     def to_dict(self):
         return {
```

Comment:
checkt what built-it method `__dict__` would return, maybe it would smth you need

Reply:
Thanks for the suggestion, at this moment, I want to keep the content under control by assigning values manually.